### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/circl/actions/runs/6066050172#summary-16456203100

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/circl/actions/runs/6066050301#summary-16456203298

### CODE REVIEW

File: abe/cpabe/tkn20/example_test.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func checkPolicy(in map[string][]string) bool {
 
 func Example() {
 	policyStr := `(occupation: doctor) and (country: US)`
-	invalidPolicyStr := `(ocupation: doctor) and (country: pacific)`
+	invalidPolicyStr := `(occupation1: doctor) and (country: pacific)`
```
Comment (jsoref) [inline abe/cpabe/tkn20/example_test.go:53]: This one is odd. I'm pretty sure the goal is to recognize something that doesn't have `occupation`, I think that `occupation_` or `occupation1` will work just as well as `ocupation` without resulting in spell checking tools complaining. Happy to drop this (or anything else).

File: dh/sidh/internal/p434/core.go [hunk 2]
```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // -----------------------------------------------------------------------------
-// Functions for traversing isogeny trees acoording to strategy. Key type 'A' is
+// Functions for traversing isogeny trees according to strategy. Key type 'A' is
```
Comment (jsoref) [inline dh/sidh/internal/p434/core.go:12]: it's possible that these are from third party libraries. I'm happy to try to send PRs to them, although I have a rather long queue and make no promises about when I'll reach them.

In any case, if the files don't originate here, if you can give me a pattern describing files I shouldn't touch, I'm happy to drop them.

File: dh/sidh/internal/p434/core.go [hunk 3]
```diff
@@ -252,7 +252,7 @@ func DeriveSecretA(ss, prv []byte, pub3Pt *[3]Fp2) {
 	// Traverse isogeny tree
 	traverseTreeSharedKeyA(&cparam, &xR)
 
-	// Calculate j-invariant on isogeneus curve
+	// Calculate j-invariant on isogenous curve
```
Comment (jsoref) [inline dh/sidh/internal/p434/core.go:255]: I'm not remotely familiar w/ this term, but Google suggests that it's a typo...

File: dh/sidh/internal/p434/core.go (hunk 2 above)
Comment (armfazh) [reply dh/sidh/internal/p434/core.go:12]: this code is generated by file /circl/dh/sidh/templates/core.gotemp

File: dh/sidh/internal/p434/core.go (hunk 3 above)
Comment (armfazh) [reply dh/sidh/internal/p434/core.go:255]: `isogenous` is the right spelling

File: dh/sidh/sike_test.go [hunk 4]
```diff
@@ -567,7 +567,7 @@ func ExampleKEM() {
 	}
 	fmt.Printf("%t\n", bytes.Equal(ssE, ssD))
 
-	// Bob performs encapsulation with Allices's public key
+	// Bob performs encapsulation with Allice's public key
```
Comment (armfazh) [inline dh/sidh/sike_test.go:570]:
```suggestion
	// Bob performs encapsulation with Alice's public key
```

**Review by armfazh — CHANGES_REQUESTED** [review]
a couple of changes only

File: dh/sidh/sike_test.go (hunk 4 above)
Comment (jsoref) [reply dh/sidh/sike_test.go]: Addressed in https://github.com/cloudflare/circl/pull/456/commits/2d44d26eb1448fd15e372f1e7b2fdc4d4efb0bcb

File: abe/cpabe/tkn20/example_test.go (hunk 1 above)
Comment (jsoref) [reply abe/cpabe/tkn20/example_test.go]: Sounds good, applied as https://github.com/cloudflare/circl/pull/456/commits/7cdef5cf5c8f2e08367e8a35b42733369a3488f3

**Review by armfazh — APPROVED** [review]

**Comment by armfazh** [comment]
Thanks @jsoref  for you contribution

**Pull request merged**